stdlib/contrib/mhall119/helpers/helpers_test.flux

```diff
@@ -0,0 +1,30 @@
+package helpers_test
+
+
+import "testing"
+import "contrib/mhall119/helpers"
+
+option testData = {testValue: 125}
+
+// Test yielding a single scalar value
+t_yieldValue = (v) =>
+    helpers.yieldValue(v:v)
+
+test _yieldValue = () =>
```

Comment:
I would follow the `testcase` examples the `test` keyword is the older style. Look here for and example https://github.com/influxdata/flux/blob/master/stdlib/math/math_test.flux#L20

Reply:
Is there documentation on how to use this? Trying to reverse engineer from examples is proving difficult for me

Reply:
Not really :( its an activity that the Flux dev team does exclusively ATM, but we are working towards making this more accessible.